src/utils/components/InstanceTypeModal/InstanceTypeModal.tsx

```diff
@@ -119,4 +122,26 @@ const InstanceTypeModal: FC<InstanceTypeModalProps> = ({
   );
 };
 
+export const ConnectedInstanceTypeModal: FC<
+  Pick<InstanceTypeModalProps, 'instanceTypeVM' | 'isOpen' | 'onClose' | 'onSubmit'>
+> = (props) => {
```

Comment:
if props are used in the component and not just passed on, we should destruct if for readability